.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
```

Comment:
will this check subfolders and find `/docs/requirements_docs.txt`? does it find `requirements.txt` at all?

Reply:
For `docs` we need an additional package manager entry

Reply:
because of the path? that could also be changed...

Reply:
and yes, it finds `requirements.txt`, but not requirements_dev.txt`. I think we can accept that, what do you think? 

(Other people are having the same question though: https://github.community/t/multiple-requirements-file-in-dependabot/190134)

Reply:
added additional package manager